ledger/sandbox-common/src/main/scala/platform/sandbox/config/package.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandbox
+
+import scalaz.{@@, Tag}
+
+package config {
+
+  sealed trait LedgerNameTag
+
+}
+
+package object config {
+
+  type LedgerName = String @@ LedgerNameTag
```

Comment:
Since this is not used in `//ledger/sandbox`, can this be moved to `sandbox-classic`?

Reply:
Good catch. I added the name to `SandboxConfig` before deciding that was a bad idea.

Reply:
Done.